### ISSUE TITLE
Update to Godot 3.3.3

### ISSUE THREAD

**Issue by HaSa1002** [Libre-TrainSim/Libre-TrainSim#135]
We should update the repo to Godot 3.3.3.
Testing locally, I found some things did break (will add as I find more)

- [x] Someone is causing that (was likely an editor bug):
	```
	_process_line: Index line = 0 is out of bounds (l.offset_caches.size() = 0).
	At: scene/gui/rich_text_label.cpp:170
	```
- [x] I did a fix for that in one of my PRs (UI probably, but isn't merged yet):
	```
	SCRIPT ERROR: _handle_delayed_calls: Invalid get index 'delay' (on base: 'Nil').
	At: res://addons/jean28518.jTools/jEssentials/jEssentials.gd:129
	```

**Comment by Jean28518** [comment]
Already using GoDot 3.3.3. 
But yeah, thorugh many changes in the code some things need to be fixed.
E.g. the pause menu also doesn't work anymore properly after adding the fullscreen shortcut... 
Thanks for mentioning the bugs!

I don't know when I will work on LibreTrainSim further on, for now I have some exams at university :)

**Comment by HaSa1002** [comment]
No problem. I am in the middle of another project for the Godot Wild Jam and at the same time there is some private stuff going on as well.
As long as you are merging PRs, we are happy :D

**Comment by HaSa1002** [comment]
Fixed by #139 and #141
